api.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type RecurseAPI struct {
+	rcAPIURL string
+}
+
+type RecurserProfile struct {
+	Stints []Stint
+}
+
+type Stint struct {
+	In_progress bool
+}
+
+func (ra *RecurseAPI) userIsCurrentlyAtRC(accessToken string, email string) bool {
```

Comment:
I realized that this endpoint is very similar to the `getCurrentlyActiveEmails` endpoint!

I think we can combine these two API calls to simplify things :D

Reply:
Done!